packages/xs-extra/xapi-storage.master/opam

```diff
@@ -13,6 +13,10 @@ depends: [
   "ocaml"
   "jbuilder" {build}
   "ounit" {with-test}
+  "alcotest" {with-test}
+  "lwt" {with-test}
+  "rpclib" {with-test}
+  "ppx_deriving_rpc" {with-test}
```

Comment:
This one looks a bit odd, why would this be a test-only dependency? If the build passes we can merge though

Reply:
I used Dune to query the dependencies and they only show up for `@runtest`